scripts/generate-release-notes.sh

```diff
@@ -161,16 +161,24 @@ else
   cat << EOF >> "$1"
 ## Changes
 
-Raw diff: [$DIFF_TAG...$RELEASE_TAG](https://github.com/litentry/litentry-parachain/compare/$DIFF_TAG...$RELEASE_TAG)
+Raw diff: [$DIFF_TAG...$RELEASE_TAG]($REPO/compare/$DIFF_TAG...$RELEASE_TAG)
 
 Details:
 
 EOF
 
-  # use %n as a workaround otherwise there's no newline after the whole body
-  git log --no-merges --abbrev-commit --pretty="format:%h|%s%n" $DIFF_TAG..$RELEASE_TAG | grep -v "^$" | while read -r f; do
+  labels=("C0-breaking" "C1-noteworthy")
+
+  git log --no-merges --abbrev-commit --pretty="format:%h|%s%n" "$DIFF_TAG..$RELEASE_TAG" | grep -v "^$" | while read -r f; do
     commit=$(echo "$f" | cut -d'|' -f1)
     desc=$(echo "$f" | cut -d'|' -f2)
-    echo -e "- [\`$commit\`]($REPO/commit/$commit) $desc" >> "$1"
+    output="- [\`$commit\`]($REPO/commit/$commit) $desc"
+    
+    for ((i=0; i<${#labels[@]}; i++)); do
+      label=$(gh pr list --search "$commit" --label "${labels[i]}" --state merged)
+      [ -n "$label" ] && output+=" $REPO/labels/${labels[i]}"
```

Comment:
Couldn't get a chance to try it. But would it be better to add some new line at the end of each line?

Reply:
The testing evidence in the description of this PR already shows the visual looks ok with line break already.